src/bam-utils.js

```diff
@@ -82,29 +113,28 @@ export const getSubstitutions = (segment) => {
         substitutions.push({
           pos: currPos,
           length: 0.1,
-          sub,
           type: 'I',
         });
       } else if (sub.type === 'D') {
         substitutions.push({
           pos: currPos,
           length: sub.length,
-          sub,
           type: 'D',
         });
         currPos += sub.length;
       } else if (sub.type === 'N') {
         substitutions.push({
           pos: currPos,
           length: sub.length,
-          sub,
+          //sub,
           type: 'N',
         });
         currPos += sub.length;
       } else if (sub.type === '=' || sub.type === 'M') {
         currPos += sub.length;
       } else {
         // console.log('skipping:', sub.type);
+        // console.log(cigarSubs)
```

Comment:
```suggestion
```

I know, I'm guilty as well but we should probably strive to get rid of useless console log statements.

Reply:
Agreed.